src/s2wasm.h

```diff
@@ -669,7 +669,7 @@ class S2WasmBuilder {
       skipComma();
       auto curr = allocator.alloc<Store>();
       curr->type = type;
-      int32_t bytes = getInt();
+      int32_t bytes = getInt()/8;
```

Comment:
A `char` is always one byte in C. `CHAR_BIT` is "number of bits for smallest object that is not a bit-field (byte)". In C++ "For any object (other than a base-class subobject) of trivially copyable type T, whether or not the object holds a valid value of type T, the underlying bytes (1.7) making up the object can be copied into an array of char or unsigned char. If the content of the array of char or unsigned char is copied back into the object, the object shall subsequently hold its original value."